.github/labeler.yml

```diff
@@ -0,0 +1,273 @@
+
+area/arc:
+ - extensions/arc/**/*
+
+area/artemis:
+ - extensions/artemis-core/**/*
+ - extensions/artemis-jms/**/*
+ - integration-tests/artemis-core/**/*
+ - integration-tests/artemis-jms/**/*
+
+area/config:
+ - extensions/config-yaml/**/*
+
+area/documentation:
+ - docs/**/*
+
+area/hibernate-orm:
+ - extensions/hibernate-orm/**/*
+ - integration-tests/hibernate-orm-panache/**/*
+
+area/hibernate-search:
+ - extensions/hibernate-search-elasticsearch/**/*
+ - integration-tests/hibernate-search-elasticsearch/**/*
+
+area/hibernate-validator:
+ - extensions/hibernate-validator/**/*
+ - integration-tests/hibernate-validator/**/*
+
+area/infinispan:
+ - extensions/infinispan-client/**/*
+ - extensions/infinispan-embedded/**/*
+ - integration-tests/infinispan-cache-jpa/**/*
+ - integration-tests/infinispan-client/**/*
+ - integration-tests/infinispan-embedded/**/*
+
+area/jaeger:
+ - extensions/jaeger/**/*
+
+area/jaxb:
+ - extensions/jaxb/**/*
+
+area/kafka:
+ - extensions/kafka-client/**/*
+ - integration-tests/kafka/**/*
+
+area/kafka-streams:
+ - extensions/kafka-streams/**/*
+ - integration-tests/kafka-streams/**/*
+
+area/kogito:
+ - extensions/kogito/**/*
+ - integration-tests/kogito/**/*
+ - integration-tests/kogito-maven/**/*
+
+area/kotlin:
+ - extensions/kotlin/**/*
+ - integration-tests/kotlin/**/*
+
+area/kubernetes:
+ - extensions/kubernetes/**/*
+ - extensions/kubernetes-client/**/*
+ - integration-tests/kubernetes/**/*
+ - integration-tests/kubernetes-client/**/*
+
+area/logging:
+ - extensions/logging-gelf/**/*
+ - extensions/logging-json/**/*
+ - extensions/logging-sentry/**/*
+ - integration-tests/logging-gelf/**/*
+
+area/maven:
+ - integration-tests/maven/**/*
+
+area/mongodb:
+ - extensions/mongodb-client/**/*
+ - integration-tests/mongodb-client/**/*
+ - integration-tests/mongodb-panache/**/*
+
+area/narayana:
+ - extensions/narayana-jta/**/*
+ - extensions/narayana-stm/**/*
+ - integration-tests/narayana-jta/**/*
+ - integration-tests/narayana-stm/**/*
+
+area/neo4j:
+ - extensions/neo4j/**/*
+ - integration-tests/neo4j/**/*
+
+area/oidc:
+ - extensions/oidc/**/*
+ - integration-tests/oidc/**/*
+ - integration-tests/oidc-code-flow/**/*
+
+area/panache:
+ - extensions/panache/**/*
+
+area/persistence:
+ - integration-tests/jpa/**/*
+ - integration-tests/jpa-derby/**/*
+ - integration-tests/jpa-h2/**/*
+ - integration-tests/jpa-mariadb/**/*
+ - integration-tests/jpa-mssql/**/*
+ - integration-tests/jpa-mysql/**/*
+ - integration-tests/jpa-postgresql/**/*
+ - integration-tests/jpa-without-entity/**/*
+ - integration-tests/common-jpa-entities/**/*
+
+area/qute:
+ - extensions/qute/**/*
+
+area/reactive:
+ - extensions/reactive-mysql-client/**/*
+ - extensions/reactive-pg-client/**/*
+ - extensions/reactive-streams-operators/**/*
+ - integration-tests/reactive-mysql-client/**/*
+ - integration-tests/reactive-pg-client/**/*
+
+area/rest-client:
+ - extensions/rest-client/**/*
+ - extensions/elasticsearch-rest-client/**/*
```

Comment:
This one shouldn't be here but with the hibernate-search label. At least for now. It's not exposed, just an internal library of the Search extension.